src/osd/OSD.cc

```diff
@@ -6020,6 +6020,12 @@ COMMAND("compact",
 COMMAND("smart name=devid,type=CephString,req=False",
         "runs smartctl on this osd devices.  ",
         "osd", "rw", "cli,rest")
+COMMAND("cache drop",
+        "Drop all OSD caches",
+        "osd", "rw", "cli,rest")
```

Comment:
Hmm actually can we make this require rwx? We don't want normal clients to be able to force a cache drop command since it's got performance implications on all the others as well.

Reply:
Ack and done.